Add tracing feature to readme example (#7)
biscuit-auth/biscuit-actix-middleware: README.md

A complete, runnable example can be found in `examples/readme.rs`, and can be run with `BISCUIT_PUBLIC_KEY=<public key> cargo run --example readme`.

> Optionally, you can enable tracing feature with `BISCUIT_PUBLIC_KEY=<public key> cargo run --example readme --features tracing` to observe middleware traces as log in the console.

Reviewer: ```suggestion
Optionally, you can enable tracing by running `BISCUIT_PUBLIC_KEY=<public key> cargo run --example readme --features tracing` to observe middleware traces as logs in the console.
```
